src/run.rs

```diff
@@ -118,23 +120,30 @@ pub(crate) async fn prepare_run_env(cfg: &PullAndRunSettings) -> Result<RunEnv>
     }
     let policy_evaluator = policy_evaluator_builder.build()?;
 
-    let req_obj = match request {
-        serde_json::Value::Object(ref object) => {
-            if object.get("kind").and_then(serde_json::Value::as_str) == Some("AdmissionReview") {
-                object
-                    .get("request")
-                    .cloned()
-                    .ok_or_else(|| anyhow!("invalid admission review object"))
-            } else {
-                Ok(request)
+    let request = if cfg.raw {
+        ValidateRequest::Raw(req_obj)
+    } else {
+        let req_obj = match req_obj.clone() {
+            serde_json::Value::Object(object) => {
+                if object.get("kind").and_then(serde_json::Value::as_str) == Some("AdmissionReview")
+                {
+                    object
+                        .get("request")
+                        .cloned()
+                        .ok_or_else(|| anyhow!("invalid admission review object"))
+                } else {
+                    Ok(req_obj)
+                }
             }
-        }
-        _ => Err(anyhow!("request to evaluate is invalid")),
-    }?;
+            _ => Err(anyhow!("request to evaluate is invalid")),
+        }?;
+        let adm_req: AdmissionRequest = serde_json::from_value(req_obj)?;
+        ValidateRequest::AdmissionRequest(adm_req)
+    };
```

Comment:
I'm not sure about this. in the policy server, we expect the raw request to be bewrapped in something like:
```json
{
"request": ...
}
```

The command accept both AdmissionReviews and the inner AdmissionRequest.
It would be nice to implement the same behavior for Raw requests but I am afraid ti could cause confusion if the users sends payloads such as:

```
{
  "request": ""
  "foo": "bar"
}
```
where the `request` key is part of the request payload itself.

Reply:
I think the code you currently wrote is fine. We just have to be explicit inside of the docs about how to use kwctl to process raw requests